sor/src/main/java/com/bazaarvoice/emodb/sor/db/test/InMemoryDataReaderDAO.java

```diff
@@ -260,7 +261,7 @@ private synchronized void update(Table table, String key, UUID changeId, Delta d
 
     @Override
     public synchronized void compact(Table table, String key, UUID compactionKey, Compaction compaction,
-                                     UUID changeId, Delta delta, Collection<UUID> changesToDelete, List<History> historyList, WriteConsistency consistency) {
+                                     UUID changeId, Delta delta, Collection<DeltaClusteringKey> changesToDelete, List<History> historyList, WriteConsistency consistency) {
         checkNotNull(table, "table");
         checkNotNull(key, "key");
         checkNotNull(compactionKey, "compactionKey");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "compactionKey" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS1192)